commonv5/ts/helpers/__tests__/api-test.ts

```diff
@@ -43,8 +43,44 @@ jest.mock("../../helpers/request", () => ({
   get: jest.fn(),
 }));
 
-beforeEach(() => {
-  (get as jest.Mock<any>).mockClear();
+describe("fetchWithRetry", () => {
+  beforeEach(() => {
+    (get as jest.Mock<any>).mockClear();
+  });
+
+  it(
+    "should not fail after up to 2 attempts",
+    async () => {
+      for (let i = 0; i < 2 - 1; i++) {
```

Comment:
NP: I assume you wanted to run 2 failures here

```suggestion
      for (let i = 0; i < 2; i++) {
```

Reply:
Thanks for your due diligence 🙏